packages/expo-modules-autolinking/src/platforms/ios.ts

```diff
@@ -96,3 +108,18 @@ function formatArrayOfClassNames(classNames: string[]): string {
   return `[${classNames.map((className) => `\n${indent.repeat(3)}${className}.self`).join(',')}
 ${indent.repeat(2)}]`;
 }
+
+/**
+ * Formats an array of module to Swift's array containing ReactDelegateHandlers
+ */
+export function formatArrayOfReactDelegateHandler(modules: ModuleDescriptor[]): string {
+  const values: string[] = [];
+  for (const module of modules) {
+    for (const handler of module.reactDelegateHandlers) {
+      values.push(`(packageName: "${module.packageName}", klass: ${handler}.self)`);
```

Comment:
I'm not a big fan of `klass` or `clazz`. Maybe just `handler`?
```suggestion
      values.push(`(packageName: "${module.packageName}", handler: ${handler}.self)`);
```